api/src/main.py

```diff
@@ -0,0 +1,51 @@
+#
```

Comment:
openapi generator still creates a main.py that is similar. Is it still used?
You should consider preventing the creation of main.py. I think it can be done in in gen-config.yaml.

Reply:
I'm looking at the CLI documentation and I don't see any option to prevent the generation of a single file, https://openapi-generator.tech/docs/generators/python-fastapi.

Reply:
Never trust chatgpt. Never mind.